examples/md-flexible/Objects.h

```diff
@@ -0,0 +1,434 @@
+#pragma once
+#include <array>
+#include <vector>
+#include "Generator.h"
+#include "autopas/utils/ArrayMath.h"
+#include "autopas/utils/ArrayUtils.h"
+
+/**Containes all Objects with their properties and functionalities for their generation in class Generator.h
+ * and information prints in the yamlParser class
+ * */
+
+class CubeGrid {
```

Comment:
Consider creating a superclass which enforces the implementation of `getBoxMin()`, `getBoxMax()`,... and stores all common properties like `mass` and `velocity`.